src/main/java/com/github/black0nion/blackonionbot/mongodb/MongoManager.java

```diff
@@ -85,18 +85,4 @@ public static void updateValue(final MongoCollection<Document> collection, final
 			collection.insertOne(updatedValue);
 		}
 	}
-
```

Comment:
Add Java doc to this class.

Reply:
I'm thinking about switching to PostgreSQL anyway, so I'll wait on this.

Reply:
ok